### DISCUSSION TITLE
New Gulp Setup

### DISCUSSION THREAD

**Discussion by momin-riyadh** [h5bp/html5-boilerplate#2951] [Q&A] [answered]
I create a new project from PhpStorm and found a issue for installing `package.json` file, and I resolve it by downgrade my node version from __18 to 14__. My questions is: how can I watch continues development change on browser I fund only build script command for gulp

**Comment by roblarsen** [comment, marked as answer]
I have no idea about what PhpStorm is doing with anything. For our part, you never used gulp for developing a web site. Gulp was only ever used to develop the PROJECT.  The current release ships with Parcel for development and the upcoming version ships with Webpack for doing development builds. There is no gulp solution and there never has been a gulp solution to develop a site or application using HTML5 Boilerplate as a baseline.
